### ISSUE TITLE
[BugFix] Add authentication for queries

### ISSUE THREAD

**Issue by F-Camuso** [EduardoGiacomini/mafcust-backend#9]
### I am submiting a: 
* [ ] Feature request; 
* [X] Bug fixes.

### Current behaviour
Currently all queries can be acessed as long as you have the endpoint. 

### Expected behaviour
Only authenticated users should have this access

### How to reproduce the bug?

just send any query to /graphql...
 
### Possible solution
Add the rule isAuthenticated to all queries that involves retrieving all data from a table
Example: 
 ```      
Query: {
        users: isAdmin,
        patients: isAuthenticated,
        examCategories: isAuthenticated,
      },
```

**Comment by EduardoGiacomini** [comment]
It's so important, we should have to protect these endpoints.

@F-Camuso @SantiagoMateus, can we implement these changes on current sprint?

**Comment by F-Camuso** [comment]
> It's so important, we should have to protect these endpoints.
> 
> @F-Camuso @SantiagoMateus, can we implement these changes on current sprint?

implemented in PR #20
